src/utils.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module wordcount/utils
+ */
+
+/**
+ * Returns plain text representation of an element and it's children. The blocks are separated by a newline (\n ).
+ *
+ * **Note:** Function walks through the entire model, which might be very spread. There should be considered throttling it during usage.
```

Comment:
KISS. It's quite obvious that it has to iterate through the entire model. And it's the consumer's responsibility to consider performance.

```suggestion
```